### ISSUE TITLE
Menu a11y issues

### ISSUE THREAD

**Issue by BorghildSelle** [equinor/energyvision#2086]
- Menu should be possible to close with Esc
- MenuButton should have aria-controls with the id of the TopbarDropdown
- TopbarDropdown should have an id

1. Tab to menu, press Enter to open
2. Press Esc to close

**Expected behavior**
Should close menu when pressing Esc

Documentation for the disclosure pattern
https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/
